include/savedata/save_table.h

```diff
@@ -10,6 +10,70 @@
 #include "struct_decls/struct_020797DC_decl.h"
 #include "struct_decls/struct_021C0794_decl.h"
 
+#define SAVE_PAGE_MAX       32
+#define SAVE_SECTOR_SIZE    (0x1000)
+
+enum SaveBlockID {
+    SAVE_BLOCK_ID_NORMAL = 0,
+    SAVE_BLOCK_ID_BOXES,
+
+    SAVE_BLOCK_ID_MAX
+};
+
+enum SaveTableEntryID {
+    SAVE_TABLE_ENTRY_SYSTEM = 0,
+    SAVE_TABLE_ENTRY_PLAYER,
+    SAVE_TABLE_ENTRY_PARTY,
+    SAVE_TABLE_ENTRY_BAG,
+    SAVE_TABLE_ENTRY_EVENTS,
+    SAVE_TABLE_ENTRY_POKETCH,
+    SAVE_TABLE_ENTRY_FIELD_PLAYER_STATE,
+    SAVE_TABLE_ENTRY_POKEDEX,
+    SAVE_TABLE_ENTRY_DAYCARE,
+    SAVE_TABLE_ENTRY_PAL_PAD,
+    SAVE_TABLE_ENTRY_MISC,
+    SAVE_TABLE_ENTRY_FIELD_OVERWORLD_STATE,
+    SAVE_TABLE_ENTRY_UNDERGROUND,
+    SAVE_TABLE_ENTRY_REGULATION_BATTLES,
+    SAVE_TABLE_ENTRY_IMAGE_CLIPS,
+    SAVE_TABLE_ENTRY_MAIL,
+    SAVE_TABLE_ENTRY_POFFINS,
+    SAVE_TABLE_ENTRY_RANDOM_GROUP,
+    SAVE_TABLE_ENTRY_JOURNAL,
+    SAVE_TABLE_ENTRY_TRAINER_CARD,
+    SAVE_TABLE_ENTRY_GAME_RECORDS,
+    SAVE_TABLE_ENTRY_BALL_SEALS,
+    SAVE_TABLE_ENTRY_CHATOT,
+    SAVE_TABLE_ENTRY_FRONTIER,
+    SAVE_TABLE_ENTRY_RIBBONS,
+    SAVE_TABLE_ENTRY_ENCOUNTERS,
+    SAVE_TABLE_ENTRY_GLOBAL_TRADE,
+    SAVE_TABLE_ENTRY_TV_BROADCAST,
+    SAVE_TABLE_ENTRY_RANKINGS,
+    SAVE_TABLE_ENTRY_WIFI_LIST,
+    SAVE_TABLE_ENTRY_WIFI_HISTORY,
+    SAVE_TABLE_ENTRY_MYSTERY_GIFT,
+    SAVE_TABLE_ENTRY_PAL_PARK_TRANSFER,
+    SAVE_TABLE_ENTRY_CONTESTS,
+    SAVE_TABLE_ENTRY_SENTENCE,
+    SAVE_TABLE_ENTRY_EMAIL,
+    SAVE_TABLE_ENTRY_WIFI_QUESTIONS,
+    SAVE_TABLE_ENTRY_PSS_BOXES,
```

Comment:
Noob question but what is this? Isn't PSS a Gen VI+ thing?

Reply:
Pokemon Storage System:

https://bulbapedia.bulbagarden.net/wiki/Pok%C3%A9mon_Storage_System

Reply:
> Pokemon Storage System:
> 
> https://bulbapedia.bulbagarden.net/wiki/Pok%C3%A9mon_Storage_System

O

What if we just called it `SAVE_TABLE_ENTRY_BOXES`?

Reply:
I'm not against it, but I think the extra clarity here is good. If anything, maybe `PCBoxes`.

Reply:
> I'm not against it, but I think the extra clarity here is good. If anything, maybe `PCBoxes`.

Sounds alright